falcon/testing/client.py

```diff
@@ -347,6 +347,11 @@ def json(self) -> Optional[Union[dict, list, str, int, float, bool]]:
 
         return json_module.loads(self.text)
 
+    def __repr__(self):
+        content_type = self.headers.get('Content-Type', '')
+
+        return 'Result<{}, {}, {}>'.format(self.status, content_type, self.content[:20])
```

Comment:
Since we are supporting CPython3.5+, I'd suggest using the `f-strings` instead of `str.format` method, does it make sense?

Reply:
Probably not yet @myusko, there is a separate issue for dropping that support -- let's take one step at a time.

Reply:
@myusko to make it clear, we do support CPython3.5+, but 3.5 does not support f-strings.
We will drop 3.5 & 3.6 support in https://github.com/falconry/falcon/issues/2049.

Reply:
@vytas7 Got it, tbh I've thought that `f-strings` was introduced in 3.5:) sorry, my bad!